test/e2e/persistent_volumes-disruptive.go

```diff
@@ -0,0 +1,263 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This test references
+// persistent_volumes.go
+// volumes.go
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api/v1"
+	metav1 "k8s.io/kubernetes/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"strings"
+)
+
+type testBody func(c clientset.Interface, f *framework.Framework, clientPod *v1.Pod, pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolume)
+type disruptiveTest struct {
+	testItStmt string
+	runTest    testBody
+}
+type kubeletOpt string
+
+const (
+	MIN_NODES                  = 2
+	NODE_STATE_WAIT            = 2 * time.Minute
+	kStart          kubeletOpt = "start"
+	kStop           kubeletOpt = "stop"
+	kRestart        kubeletOpt = "restart"
+)
+
+var _ = framework.KubeDescribe("PersistentVolumes:Disruptive", func() {
+
+	f := framework.NewDefaultFramework("disruptive-pv")
+	var (
+		c                         clientset.Interface
+		ns                        string
+		nfsServerPod              *v1.Pod
+		nfsPVconfig               persistentVolumeConfig
+		nfsServerIP, clientNodeIP string
+		clientNode                *v1.Node
+	)
+
+	nfsServerConfig := VolumeTestConfig{
+		namespace:   v1.NamespaceDefault,
+		prefix:      "nfs",
+		serverImage: "gcr.io/google_containers/volume-nfs:0.7",
```

Comment:
@jsafrane Okay.  Since there's also gcr.io images for glusterfs, rbd, ceph, and iscsi,  I think a volumeTestConfig constructor would the route to take.  As a param, it can take a `const <image path> string` along with other relevant fields.  Will work that out and push it.